Example/babel.config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
-  plugins: ['../plugin'],
+  plugins: [['../plugin', {processNestedWorklets: true}]],
```

Comment:
We have never documented this plugin option (processNestedWorklets). Should we?

Reply:
Yes, it was added in #4332 and should be documented in a separate PR.

Reply:
Let's be nice guys and write the docs for it first. I could do that if you want.

Reply:
Please do 🙏